Automattic/woocommerce-payments
# Removes Sofort from settings and checkout

#### Changes proposed in this Pull Request


In line with Sofort's permanent deprecation, this PR removes Sofort as an available payment method from settings and checkout. Mirrors related changes in #9055. Please see paJDYF-aMc-p2#comment-27057 for more information. 






#### Testing instructions


1. Before pulling these changes, remain on `develop` and ensure you have some completed orders using Sofort as a payment method. (Sofort has been soft-deprecated for all users already, so you may need to alter the response from the server to make the Sofort capability available.)
2. Apply the changes from this PR. You will also need to bump the version of the plugin to anything greater than the current version (`8.8.0`).
3. Ensure that Sofort is no longer present in the following places: in payments settings, at checkout as a payment method, and in any onboarding tasklists, in payment methods list when disabling WooPayments from settings.
4. Ensure that the following functionality still persists: orders completed with Sofort still contain Sofort icons and payment information on order pages, transactions list, and transaction details, merchant is still able to refund purchases completed with Sofort.

Note that Sofort is no longer present in our critical testing flows since our soft deprecation, so this will not need to be updated.

-------------------


**Post merge**
